src/PHPMailer.php

```diff
@@ -3380,7 +3383,7 @@ protected function cidExists($cid)
     public function inlineImageExists()
     {
         foreach ($this->attachment as $attachment) {
-            if ($attachment[6] == 'inline') {
+            if ($attachment[6] === 'inline') {
```

Comment:
This (and the next one) should be yoda-flipped

Reply:
done, also more yoda.

altho my personal opinion it's counter to intuitive. forces one reading left to right to start reading from right to left.

and if it's intended to be protection against accidential assign, there's no correct way to solve if both sides are non-constants, you can still cause wrong behaviour by using wrong operator:

```php
                return $params[0] = $kind;
```

Reply:
I agree, and argued much the same on Symfony's code style recently - they argued you should do it everywhere, without exception, even where it made no difference! IMHO, the only real place worth living with the readability issue is in conditions with one constant - the payoff is that case is that inadvertent assignment becomes a compile-time error rather than a runtime mystery, and that's worth having.

Reply:
having compile time error vs runtime error - it's already to late, the file is in production.

code quality and behavior should be tested with unit/integration-tests. broken code should never be merged to default branch (`master`) :)

Reply:
Of course, but it's all about reducing accidents, and compile-time errors are far easier to deal with whenever they happen, whether in dev or production, so I think yoda comparisons are a good defensive habit from that POV. Aside from that they are quite annoying...